styles/globals.css

```diff
@@ -1,5 +1,26 @@
 :root {
-  --space: 1rem;
+  --ratio: 1.5;
+  --space-5: calc(var(--space-4) / var(--ratio));
+  --space-4: calc(var(--space-3) / var(--ratio));
+  --space-3: calc(var(--space-2) / var(--ratio));
+  --space-2: calc(var(--space-1) / var(--ratio));
+  --space-1: calc(var(--space0) / var(--ratio));
+  --space0: 1rem;
+  --space1: calc(var(--space0) * var(--ratio));
+  --space2: calc(var(--space1) * var(--ratio));
+  --space3: calc(var(--space2) * var(--ratio));
+  --space4: calc(var(--space3) * var(--ratio));
+  --space5: calc(var(--space4) * var(--ratio));
+  --space6: calc(var(--space5) * var(--ratio));
```

Comment:
I'm a little confused by the naming decisions here. In some cases there is kebab case while in others we are not. Is there a reason for this? Otherwise it would be good for these to be consistent and as per the [Naming guide](https://github.com/the-bax/new-jac-city/tree/main/guide/naming#guides-for-naming) they should be camel case :grin:

Reply:
Oh, I see, it is bad naming. `--space-1` should have meant a smaller space, and `--space1` means a larger space. What about: 
```js
--enlargedSpace1: calc(var(--space0) * var(--ratio));
--shrinkedSpace1: calc(var(--space0) / var(--ratio));
--space: 1rem;
```

Reply:
Still not very clear. I don't see any reason for the `space0` if it duplicates `space`. I would just have one:

```css
--defaultSpace: 1rem;
--spaceRatio: 1.5;
```

Then be clear:

```css
--spaceFiveRatiosSmaller: calc(var(--spaceFourRatiosSmaller) * var(--spaceRatio));
--spaceFourRatiosSmaller: calc(var(--spaceThreeRatiosSmaller) * var(--spaceRatio));
--spaceThreeRatiosSmaller: calc(var(--spaceTwoRatiosSmaller) * var(--spaceRatio));
--spaceTwoRatiosSmaller: calc(var(--spaceOneRatioSmaller) * var(--spaceRatio));
--spaceOneRatioSmaller: calc(var(--defaultSpace) * var(--spaceRatio));

--spaceOneRatioBigger: calc(var(--defaultSpace) * var(--spaceRatio));
--spaceTwoRatiosBigger: calc(var(--spaceOneRatioBigger) * var(--spaceRatio));
--spaceThreeRatiosBigger: calc(var(--spaceTwoRatiosBigger) * var(--spaceRatio));
--spaceFourRatiosBigger: calc(var(--spaceThreeRatiosBigger) * var(--spaceRatio));
--spaceFiveRatiosBigger: calc(var(--spaceFourRatiosBigger) * var(--spaceRatio));
--spaceSixRatiosBigger: calc(var(--spaceFiveRatiosBigger) * var(--spaceRatio));
```

Or something like that that tells you precisely what it is. Examples:

```css
--fiveRatiosSmallerSpace
--smallerSpaceByFiveRatios
```

Reply:
I am also very hesitant when naming the colors. 
Should I name it as the color's name, like `--darkBlue`, or where the color been used, like `--linkColor`?

Reply:
My opinion is that you should avoid the name of the color except as a theme, maybe:

```css
--darkBlue: #000066;

--linkColor: var(--darkBlue);
```

This way in the CSS you use `linkColor` but you can look at the variable declarations and see immediately what the colors are. And you only have to change them in one spot.